C++/1_Learn_the_basics/1.5_Learn_Basic_Recursion/7_ReverseArray.cpp

```diff
@@ -1,24 +1,29 @@
-// https://bit.ly/3QugUVo
-
 #include<bits/stdc++.h>
 using namespace std;
 
-int main(){
-    int n;
-    cin >> n;
-    int a[n];
-    for(int i = n-1; i >= 0; i--){
-        cin >> a[i];
+void rev(int i,int a[],int n)
```

Comment:
Could you please change the function name to something more intuitive something like `reverseArray` or `reverse`.